scenarios5/20_Southern_Guardian_Room.cfg

```diff
@@ -173,21 +268,41 @@
                 [movement_costs]
                     flat={UNREACHABLE}
                 [/movement_costs]
+#ifdef HARD
+                                   [effect]
+                                        apply_to=new_ability
+                                        [abilities]
+                                            {ABILITY_ALL_DAMAGE_AURA 30}
```

Comment:
Are you sure you wanted this one? This one increases also the damage of the user's enemies (that is, the players').

Reply:
No, don't want that.

That must be the new AMLA (elf sorc line?) that I never take.